openstack/identity/v3/extensions/tokens/requests.go

```diff
@@ -0,0 +1,293 @@
+package tokens
+
+import (
+	"net/http"
+
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/openstack/identity/v3/extensions"
+)
+
+// Scope allows a created token to be limited to a specific domain or project.
+type Scope struct {
+	ProjectID   string
+	ProjectName string
+	DomainID    string
+	DomainName  string
+	TrustID     string
```

Comment:
If this request is only used by trust extension, ProjectID/ProjectName/DomainID/DomainName scope is not needed.